feature/home/src/main/java/com/yapp/feature/home/component/HomeAttendanceContent.kt

```diff
@@ -99,10 +82,15 @@ private fun UpcomingSessionCard(
     ) {
         Text(
             text = stringResource(R.string.session_today_text, today.format(DATE_OUTPUT)),
-            style = YappTheme.typography.label2Medium,
+            style = YappTheme.typography.headline1Bold,
             color = YappTheme.colorScheme.labelNormal
         )
 
+        HomeAttendanceNotice(
+            isNotToday = parsedDate?.let { today.isEqual(it) } ?: true,
+            upcomingSession = session
+        )
+
         YappSolidPrimaryButtonLarge(
             modifier = Modifier.fillMaxWidth(),
             text = parsedDate?.let {
```

Comment:
```suggestion
if (session != null) { 
        YappSolidPrimaryButtonLarge(
            modifier = Modifier.fillMaxWidth(),
            text = parsedDate?.let {
```

<img width="404" height="537" alt="image" src="https://github.com/user-attachments/assets/e6a7354b-79a6-45e4-9568-935f823bed8e" />

R: 다음 세션 정보가 없다면 `M월 D일 세션 예정`, `다가오는 세션이 없어요` 를 표시 안하기 때문에 session(=upcomingSession) 이 null이 아닐때만 표시하게 수정해야 할 것 같아요

Reply:
[1e47f0f](https://github.com/YAPP-admin/yappu-world-android/pull/226/commits/1e47f0f7b6dd01d0491868636e7e009d9d172e54)

이거는 UpcomingSessionCard 가 역할이 점점 커지는거 같아 아예 로직 분리 처리를 호이스팅 시켜서 
캡쳐해주신 세션 카드 컴포넌트를 만들어서 구분 처리했어요